website/docs/basics/blocks-wrapper.md

```diff
@@ -5,3 +5,91 @@ sidebar_label: Wrapper
 ---
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
+
+The wrapper is our best and most powerful feature. In a nutshell, the wrapper is nothing more than a high order component (parent component). 
+
+When setting up block registration, we made, so every block view passes the true wrapper component before it is registered. This gives us the ability to have multiple shared attributes set in one component. The wrapper is designed to be the ultimate top-level component that controls how your block behaves in the website layout. By default, you can control a whole lot of stuff, but there is an ability to add your custom attributes and fine-tune wrapper to your project needs. Here are some of the features:
```

Comment:
```suggestion
When setting up block registration, we made it so that every block view (PHP part) passes through the wrapper component before it is registered. This gives us the ability to set multiple shared attributes in one component.

The wrapper is designed to be the ultimate top-level component that controls how your block behaves in the website layout. The wrapper is a sort of a 'section' in traditional builders. By default, you can control a whole lot of stuff, but there is an ability to add your custom attributes and fine-tune wrapper to your project needs. Here are some of the features:
```